docs/resources/resource.md

```diff
@@ -70,6 +70,7 @@ resource "twingate_resource" "resource" {
 - `is_browser_shortcut_enabled` (Boolean) Controls whether an "Open in Browser" shortcut will be shown for this Resource in the Twingate Client.
 - `is_visible` (Boolean) Controls whether this Resource will be visible in the main Resource list in the Twingate Client.
 - `protocols` (Block List, Max: 1) Restrict access to certain protocols and ports. By default or when this argument is not defined, there is no restriction, and all protocols and ports are allowed. (see [below for nested schema](#nestedblock--protocols))
+- `security_policy_id` (String) The ID of a `twingate_security_policy` to set as this Resource's Security Policy.
```

Comment:
Can we also add an example on how to use this resource under Example Usage?

For example:
data "twingate_security_policy" "test_policy" {
  name = "test policy"
}

resource "twingate_resource" "resource" {
...
  security_policy_id = data.twingate_security_policy.test_policy.id

Reply:
added example with datasource